pkg/workloadapi/workload.proto

```diff
@@ -0,0 +1,101 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package proto;
+option go_package="pkg/workloadapi";
+
+message Workload {
```

Comment:
Can we define this proto in a standard repo, api, for example? Or at least give it some thought since it's clearly in a draft state.

Reply:
We discussed this for about 15min in the working group - it is a draft. This is being merged into the experimental branch, not master. Once it's not a draft it will be in api